go/v4/exchange.go

```diff
@@ -212,7 +212,8 @@ func (this *Exchange) InitParent(userConfig map[string]interface{}, exchangeConf
 		Transport: transport,
 	}
 
-	if IsTrue(IsTrue(this.SafeBool(userConfig, "sandbox")) || IsTrue(this.SafeBool(userConfig, "testnet"))) {
+	userOptions := this.SafeDict(userConfig, "options")
+	if IsTrue(IsTrue(this.SafeBool(userOptions, "sandbox")) || IsTrue(this.SafeBool(userOptions, "testnet"))) {
```

Comment:
@caoilainnl actually we read it from the 2 places (yeah a bit weird)

- one here from the userConfig directly
- from options inside afterConstruct
<img width="1454" height="278" alt="image" src="https://github.com/user-attachments/assets/cf9ac38f-657e-4a58-913c-6cb1859951a4" />